block/cardano-sl-block.cabal

```diff
@@ -175,3 +175,26 @@ Benchmark bench-block
                    , formatting
                    , universum
                    , deepseq
+
+test-suite block-test
+  main-is:          Spec.hs
+  default-language: Haskell2010
+  type:             exitcode-stdio-1.0
+  build-depends:    base
+                  , universum
+                  , hspec
+                  , text
+                  , bytestring
+                  , cardano-sl-core
+                  , cardano-sl-block
+                  , cardano-sl-db
+                  , cardano-sl-delegation
+                  , cardano-sl-txp
+                  , cardano-sl-ssc
+                  , cardano-sl-infra
+                  , cardano-sl-crypto
+                  , cardano-sl-update
+                  , lens
+                  , cardano-sl-util
+                  , cardano-sl-binary
+  hs-source-dirs:   test .
```

Comment:
Having the test module reload/recompile all code in the test suite is not a good practice; it breaks `hpc` and related tooling IIRC.